<br>
<table style="width:98%;margin-left:1%;">
<tr height=40><td colspan=3 align=left><h5>Frequently asked questions for GlyGen</h5></td></tr>
<tr>
<td width=10>1.</td>
<td class=faqquestion id=question_1>
What does protein-centric data include?
</td>
</tr>
<tr>
<td weight=bold id=answer_1 class=faqanswer colspan=2>
Protein-centric data include data types and information that is about some particular protein coding gene, or that can be mapped to the canonical protein sequence representing that gene. Examples include pathways, Gene Ontology, localization, etc.
</td>
</tr>

<tr>
<td>2.</td>
<td id=question_2 class=faqquestion>
What does proteoform-centric data include?</td>
</tr>
<tr>
<td id=answer_2 class=faqanswer colspan=2>
Proteoform-centric data include data types and information that is about some particular proteoform or that pertains to a specific amino acid site. A proteoform is the specific protein product of a gene; multiple proteoforms might arise due to differences in genetic variation, alternative splicing or translation start site selection, or post-translational modifications.
</td>
</tr>

<tr>
<td>3.</td>
<td id=question_3 class=faqquestion>
What does glycan-centric data include?
</td>
</tr>
<tr>
<td id=answer_3 class=faqanswer colspan=2>
Glycan-centric data include data types and information that is about specific glycans.
</td>
</tr>

<tr>
<td>4.</td>
<td id=question_4 class=faqquestion>
Is it possible to have the same data or information mapped to proteins, glycans and proteoforms?
</td>
</tr>
<tr>
<td id=answer_4 class=faqanswer colspan=2>
Yes.
</td>
</tr>


<tr>
<td>5.</td>
<td id=question_5 class=faqquestion>
Is it possible to have datasets that are neither protein, proteoform or glycan centric?
</td>
</tr>
<tr>
<td id=answer_5 class=faqanswer colspan=2>
Yes, such datasets are listed under "other" category.
</td>
</tr>

<tr>
<td>6.</td>
<td id=question_6 class=faqquestion>
What is the format of the readme files?
</td>
</tr>
<tr>
<td id=answer_6 class=faqanswer colspan=2>
All readme files follow the BioCompute specifications. Technical specifications can be found at <a href="https://hive.biochemistry.gwu.edu/htscsrs/">https://osf.io/r6s4u/.</a>
</td>
</tr>

<tr>
<td>7.</td>
<td id=question_7 class=faqquestion>
How is data integrated into GlyGen?
</td>
</tr>
<tr>
<td id=answer_7 class=faqanswer colspan=2>
<img src = "content/data.png">
</td>
</tr>

<tr>
<td>8.</td>
<td id=question_8 class=faqquestion>
What licence does GlyGen use?
</td>
</tr>
<tr>
<td id=answer_8 class=faqanswer colspan=2>
The GlyGen is licensed under CC BY 4.0. You are free to share (copy and redistribute the material in any medium or format) and adapt (remix, transform, and build upon the material) for any purpose, even commercially.<br> You must give appropriate credit (by referring to GlyGen for the whole resource or Object ID for individual datasets).<br> You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.<br>
Datasets made available through GlyGen includes archival of versions. We recommend authors to use Creative Commons Attribution licence (CC BY) for all versions when they share their datasets with GlyGen.<br> This means that users of GlyGen are entitled to use, reproduce, disseminate or display these datasets provided the original authors and GlyGen is attributed.
</td>
</tr>

<tr>
<td>9.</td>
<td id=question_9 class=faqquestion>
What is a proteoform?
</td>
</tr>
<tr>
<td id=answer_9 class=faqanswer colspan=2>
The term "proteoform" designates "all of the different molecular forms in which the protein product of a single gene can be found, including changes due to genetic variations, alternatively spliced RNA transcripts and post-translational modifications." (Smith et al. Nat Methods. 2013 Mar;10(3):186-7. PMID:23443629).
<br>A protein can have many different glycosolyated proteoforms that differ from each other in a number of ways: the site(s) that are glycosylated, the glycan structure at each modified site, and the presence of any other modifications. For example, the figure below shows several different glycosylated proteoforms of the sex hormone binding globulin (SHBG) that are represented in the Protein Ontology (PRO). These forms have an OGalNAc modification at Thr36 and one of several different N4GlcNAc modifications at Asn380 or Asn396.
<br><img src = "content/proteoform_faq9.png">
</td>
</tr>

<tr>
<td>10.</td>
<td id=question_10 class=faqquestion>
Why is the representation of proteoforms useful?
</td>
</tr>
<tr>
<td id=answer_10 class=faqanswer colspan=2>
Many aspects of protein function-activity, sub-cellular localization, and interaction partners, for example-are influenced by the precise combination of modifications on the protein. When proteins are represented and annotated in knowledge resources without regard to modification state, these distinctions can be lost. In contrast, proteoform-level representation makes it possible to associate annotation unambiguously with the most relevant protein form. For example, the figure below shows the relative abundances of 12 different glycosylated proteoforms of SHBG (six different glycan structures at two different sites, Asn380 and Asn396). This abundance information was associated with the corresponding proteoform terms in PRO (Protein Ontology).
<br><img src = "content/proteoform_faq10.png">
</td>
</tr>

<tr>
<td>11.</td>
<td id=question_11 class=faqquestion>
Given that proteins can have many potential glycosylation sites and there are many different glycan forms, the number of possible proteoforms quickly becomes unmanageably large. <br>How can we possibly represent proteoforms in a useful way?
</td>
</tr>
<tr>
<td id=answer_11 class=faqanswer colspan=2>
The policy of the Protein Ontology is to only represent proteoforms that have been experimentally observed. For example, if glycosylation is observed on two different sites, each of the singly modified forms can be represented in PRO; however, the proteoform with glycosylation on both sites will not be represented unless there is solid experimental evidence for its existence. In this way, the number of proteoforms per protein does not grow too large, and the forms represented are those that are of highest interest to biologists.
</td>
</tr>
</table>




